doc/main.html

```diff
@@ -148,50 +150,103 @@ <h4>Overall structure</h4>
+              <td>Mandatory (generated automatically)</td>
+            </tr>
+            <tr>
+              <td>Unnumbered</td>
+              <td><a>Foreward</a></td>
+              <td>Mandatory</td>
+            </tr>
+            <tr>
+              <td>Unnumbered</td>
+              <td><a>Introduction</a></td>
+              <td>Optional</td>
             </tr>
             <tr>
               <td>1</td>
-              <td><a>scope section</a></td>
+              <td><a>Scope</a></td>
+              <td>Mandatory</td>
             <tr>
-              <td>0..1</td>
-              <td><a>conformance section</a></td>
+              <td>2</td>
+              <td><a>Conformance</a></td>
+              <td>Mandatory</td>
             </tr>
             <tr>
-              <td>0..1</td>
-              <td><a>normative references section</a></td>
+              <td>3</td>
+              <td><a>Normative references</a></td>
+              <td>Mandatory</td>
             <tr>
-              <td>0..1</td>
-              <td><a>terms and definitions section</a></td>
+              <td>4</td>
+              <td><a>Terms and definitions</a></td>
+              <td>Mandatory</td>
             </tr>
             <tr>
-              <td>0..*</td>
-              <td><a>prose section</a></td>
+              <td>5-*</td>
+              <td>* (<a>Prose section</a>)</td>
+              <td>Mandatory</td>
             </tr>
             <tr>
-              <td>0..*</td>
-              <td><a>annex section</a></td>
+              <td>Annex A-*</td>
+              <td>* (<a>Annex section</a>)</td>
+              <td>Optional</td>
             </tr>
             <tr>
-              <td>0..1</td>
-              <td><a>bibliography section</a></td>
+              <td>Unnumbered</td>
+              <td><a>Bibliography</a></td>
+              <td>Conditional (if informative references present)</td>
             </tr>
           </tbody>
         </table>
 
+        <p class="note">For further section infomation and requirements outside of this document, see <a href="#bib-iso-directives-part2"></a> and <a href="#bib-smpte-ag-16"></a></p>
+
+      </section>
+
+      <section id="sec-toc-section">
+        <h4>Table of contents section</h4>
+
+        <p>The <dfn>Table of contents</dfn> contains a listing of all sections within the document and links to those sections.</p>
+
+        <p>No user text is required or allowed.</p>
+
+        <p>The heading and entire section will be generated automatically.</p>
+
+      </section>
+
+      <section id="sec-foreward-section">
+        <h4>Forward section</h4>
+
+        <p>The <dfn>Foreward</dfn> contains background info about SMPTE, copyright notices, and additional infomation based on the document type (<code>pubType</code>).</p>
+
+        <p>User text for additional infomation relevant to the document is allowed in addition to the general boilerplate. If adding additional text, the section shall be present as:</p>
```

Comment:
```suggestion
        <p>It can also contain author-specified text, e.g. list of substantive changes since the last edition. Author-specified text is added as follows:</p>
```

Reply:
Much better wording, thanks!